.rubocop.yml

```diff
@@ -103,7 +103,7 @@ Style/GlobalVars:
     - 'spec/**/*'
     - 'spec/jobs/event_jobs_spec.rb'
     - 'lib/generators/sufia/templates/config/redis_config.rb'
-    - 'lib/sufia/redis_event_store.rb'
+    - 'app/models/concerns/sufia/redis_event_store.rb'
```

Comment:
Moved back to `lib/` and pushed. This needs to be merged before this PR can go in: https://github.com/projecthydra-labs/curation_concerns/pull/550